app/src/main/java/com/scouts/kitchenplaner/model/usecases/DisplayIngredientList.kt

```diff
@@ -39,22 +51,33 @@ class DisplayIngredientList (
                 addIngredients(result, recipe, numberOfPeople, slot)
                 recipes.second.forEach {
                     val alternative = recipeRepository.getRecipeById(it.id).first()
-                    addIngredients(result, alternative, 1, slot) //TODO - figure out how many people are actually eating
+                    addIngredients(
+                        result,
+                        alternative,
+                        1,
+                        slot
+                    ) //TODO - figure out how many people are actually eating
                 }
             }
         }
         return result
     }
 
     @OptIn(DomainLayerRestricted::class)
-    private fun addIngredients(ingredientList: IngredientList, recipe: Recipe, numberOfPeople: Int, slot: MealSlot) {
+    private fun addIngredients(
+        ingredientList: IngredientList,
+        recipe: Recipe,
+        numberOfPeople: Int,
+        slot: MealSlot
+    ) {
         recipe.ingredientGroups.forEach {
             it.ingredients.forEach { ingredient ->
                 ingredientList.addIngredient(
                     Ingredient(
                         ingredient.name,
                         ingredient.amount * (numberOfPeople.toFloat() / recipe.numberOfPeople),
```

Comment:
Replace with ```toDouble``` (or open a separate issue as this really doesn't belong to this one, I just think that would be a bit of an overkill)